cfgov/v1/templates/cdnadmin/index.html

```diff
@@ -10,13 +10,13 @@
     {% include "wagtailadmin/shared/header.html" with title="CDN Manager" icon="tag" %}
 
     <div class="nice-padding">
-        <h2>Flush Akamai</h2>
+        <h2>Cache Invalidator</h2>
 
     {% if user_can_purge %}
         <div class="help-block help-info">
             <p>
                 Enter a full URL (e.g., <code>https://www.consumerfinance.gov/about-us/</code>)
```

Comment:
```suggestion
                 Enter a full URL (e.g., <code>https://www.consumerfinance.gov/about-us/</code>
                 or <code>https://files.consumerfinance.gov/f/documents/example.pdf</code>)
```

Reply:
There may be a weird spacing thing going on here related to GH's suggested change feature. Please confirm that the indentation remains at 16 spaces once/if the suggested change is accepted.

Reply:
So weird that the spacing worked out like that. Fixed in ff8ebe3.